Events/freshman.js

```diff
@@ -29,5 +37,12 @@ module.exports = {
             iteration++; 
         }
     }
+    async function startTimeout(message, interactionMessage) {
+        setTimeout(() => {
+            // find a way to recieve signal from buttonHandler.js to stop timeout
+            message.delete();
+            interactionMessage.delete();
+        }, 10000);
+    }
```

Comment:
If you are trying to auto delete your message if you do not respond then do this: 
In your function for handling button presses, at the end of it, you are not deleting the buttons (you are rn and you shouldn't) after deleting the buttons (message.edit({ embeds: [what ever], components: [] })). within your set timeout function, you should still have the message id of the response message or you could get it if you don't, check to see if there are still buttons on that message. If there are, then do whatever, if not, move on.

Reply:
@WindowsVistaisCool